### ISSUE TITLE
NVDA announces "Disconnected" twice when controlling another computer

### ISSUE THREAD

**Issue by hdzrvcc0X74** [nvaccess/nvda#18692]
When the connection is terminated while we are connected and controlling another computer, the "Disconnected" message is spoken twice.

### Steps to reproduce:

1. Press NVDA+Alt+R (or your custom gesture) to open the connection dialog. Disconnect if a connection is already established.
2. Set Mode to Control another computer.
3. Enter the host and key, then connect. (No other party needs to be connected for this issue to occur.)
4. Disconnect using NVDA+Alt+R (or your custom gesture).
5. You will hear the message "Disconnected" twice.

### Actual behavior:

NVDA announces the "Disconnected" message twice when controlling another computer and disconnecting via a gesture.

### Expected behavior:

NVDA should announce the "Disconnected" message only once when controlling another computer.

### NVDA logs, crash dumps and other attachments:

[nvda-remote.log](https://github.com/user-attachments/files/21754130/nvda-remote.log)

#### NVDA installed/portable/running from source:

Installed.

#### NVDA version:

NVDA alpha-51856,e038efac

#### Windows version:

Windows 11 24H2 (AMD64) build 26100.4770

#### Name and version of other software in use when reproducing the issue:

N/A

#### Other information about your system:

N/A

#### Does the issue still occur after restarting your computer?

Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No.

#### If NVDA add-ons are disabled, is your problem still occurring?

Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes.

**Comment by tareh7z** [comment]
What I forgot to mention is that the problem can also be reproduced by disconnecting via NVDA menu → Tools → Remote → Disconnect.
